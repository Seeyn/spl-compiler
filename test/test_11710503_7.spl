float foo(){
    return 1;
}

int main(){
    int a,b;
    a = 0;
    b = 0;
    1 = a + b;
}