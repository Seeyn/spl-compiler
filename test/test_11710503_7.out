Error type 8 at Line 2: incompatiable return type
Error type 6 at Line 9: left side in assignment is rvalue
